oprheum-game/InstructionOverlayNode.swift

```diff
@@ -0,0 +1,90 @@
+//  InstructionOverlayNode.swift
+//
+//  Created by Clay Reimann
+//
+
+import SpriteKit
+
+class InstructionOverlayNode: SKNode {
+    let label1Height: CGFloat = 500
+    let label2Height: CGFloat = 300
+    let continueHeight: CGFloat = 50
+
+    var size = CGSize.zero {
+        didSet(oldSize) {
+            background.path = backgroundPath()
```

Comment:
`background` is the grey overlay on which the text is displayed.
`backgroundPath` is the function that computes what `background`'s shape is